axum-macros/tests/typed_path/fail/missing_capture.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0027]: pattern does not mention field `id`
+ --> tests/typed_path/fail/missing_capture.rs:5:14
+  |
+5 | #[typed_path("/users")]
+  |              ^^^^^^^^ missing field `id`
+  |
+help: include the missing field in the pattern
+  |
+5 | #[typed_path("/users" { id })]
+  |                       ++++++
```

Comment:
This is a weird error message. I thought this case would be caught by the proc-macro directly instead of later by the compiler trying to instantiate `MyPath`.